packages/app/cypress/e2e/runner/aliases.cy.ts

```diff
@@ -0,0 +1,15 @@
+import { loadSpec } from './support/spec-loader'
+
+describe('Aliases', () => {
```

Comment:
This test is pretty heavy when we could be validating log attributes within a driver test

Reply:
Similar to this test: https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/e2e/commands/net_stubbing.cy.ts#L3531-L3532

Reply:
I am not super familiar with the driver but the way I understand it, the run time aliases that are set in the callback of intercept can't be triggered via a command? Am I missing something @emilyrohrbough ?

Reply:
Thank you! Just pushed a change